### PR TITLE
Added remove_invalid_parentheses.py

### PR DESCRIPTION
**Pull request by pratham3012**
### Describe your change:

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: backtracking/remove_invalid_parentheses.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    remove the same number of brackets to print all of them.
+
+    GeeksforGeeks: https://www.geeksforgeeks.org/remove-invalid-parentheses/
+
+    Below is the Python3 implementation of the above problem.
+"""
+
+"""
+    'is_parenthesis' Method checks if character is parenthesis(open or closed)
+"""
```
Comment (CaedenPH) [inline backtracking/remove_invalid_parentheses.py:16]:
```suggestion
In this problem, an expression is specified that can
contain opening and closing parentheses and optionally
some characters. There are no other operators for strings.
A minimum number of parentheses must be removed to make the
input string valid. If multiple valid outputs are possible,
remove the same number of brackets to print all of them.

GeeksforGeeks: https://www.geeksforgeeks.org/remove-invalid-parentheses/

:meth:`is_parenthesis` checks if character is parenthesis(open or closed)
"""
```

File: backtracking/remove_invalid_parentheses.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+
+def is_parenthesis(c):
```
Comment (CaedenPH) [inline backtracking/remove_invalid_parentheses.py:19]: What is the type of `c`?

File: backtracking/remove_invalid_parentheses.py [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    return (c == "(") or (c == ")")
```
Comment (CaedenPH) [inline backtracking/remove_invalid_parentheses.py:20]:
```suggestion
    return c in ["(", ")"]
```

File: backtracking/remove_invalid_parentheses.py [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+    temp = 0
+    level = 0
+
+    # pushing given as starting node into queue
+    q.append(str)
+    visit.add(str)
+    while len(q):
+        str = q[0]
+        q.pop()
+        if isValidString(str):
```
Comment (CaedenPH) [inline backtracking/remove_invalid_parentheses.py:63]: ?

File: backtracking/remove_invalid_parentheses.py [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+    # queue to maintain BFS
+    q = []
+    temp = 0
+    level = 0
+
+    # pushing given as starting node into queue
+    q.append(str)
+    visit.add(str)
+    while len(q):
+        str = q[0]
```
Comment (CaedenPH) [inline backtracking/remove_invalid_parentheses.py:61]: Don't use builtins as variable names

**Review by CaedenPH — CHANGES_REQUESTED** [review]
Overall, the code is extremely messy and inconsistent and doesn't run correctly.

**Comment by cclauss** [comment]
In preparation for that denial-of-service attack that is [Hacktoberfest](https://hacktoberfest.com), I am closing all pull requests that have failing tests.

**Pull request closed without merging**